app/templates/teachers/teacher-student-view.jade

```diff
@@ -364,15 +364,18 @@ mixin breadcrumbs
                               .small!= marked(solution.description || '')
                           tr
                             td
-                              if view.levelStudentCodeMap[level.get('original')]
+                              - var multipleCodes = view.levelStudentCodeMap[level.get('original')].length > 1
+                              unless view.levelStudentCodeMap[level.get('original')].length
+                                i.small(data-i18n="teacher.no_code_yet")
+                              each codemap, index in view.levelStudentCodeMap[level.get('original')]
+                                if multipleCodes
+                                  i.small(data-i18n=index%2?"ladder.ogres":"ladder.humans")
```

Comment:
Is there any guarantee that this will be the right ordering (humans, then ogres)? Reading the code, it seems like we add the code in order of sessions fetched in TeacherStudentView, which could have `["ogres", "humans"]`, `["humans", "ogres"]`, `["humans"]`, `["ogres"]`, `["ogres", "ogres", "humans"]` in some weird case, etc.

Reply:
great catch! didn't think about this before. i'll see is there another key can return the correct team